ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -1850,6 +1850,15 @@ router_pim_sparse_mode:
         other_anycast_rp_addresses:
           < ip_address_other_anycast_rp_1 >:
             register_count: < register_count_nb >
+  vrfs:
+    - name: < vrf_name >
+      ipv4:
+        rp_addresses:
+          - address: < rp_address_1 >
+            groups:
+              - < group_prefix_1/mask >
+              - < group_prefix_2/mask >
+          - address: < rp_address_2 >
```

Comment:
Please remove this line. It might confuse the people sometimes. Like they might specify only address assuming groups is optional.

Reply:
Done